R/coord-funs.R

```diff
@@ -304,3 +324,23 @@ coord_pol_centroids <- function(x, lat = NULL, lon = NULL, drop = TRUE) {
 #   coordinates(wc) <- ~long + lat
 #   wc
 # }
+
+#' @export
+#' @rdname coords
+coord_uncertain <- function(x, coorduncertainityLimit = 30000, drop = TRUE, ignore.na = FALSE){
+  if(!("coordinateUncertaintyInMeters"  %in%  names(x))){
+    stop(" 'coordinateuncertainityInMeters' variable is missing", call. = FALSE)
+  }
+  if(ignore.na) x <- x[!is.na(x$coordinateUncertaintyInMeters), ]
+  uncertain_indices <- which(x$coordinateUncertaintyInMeters > coorduncertainityLimit)
+  uncertain <- x[uncertain_indices, ]
+  if (NROW(uncertain) == 0) uncertain <- NA
+  if(drop){
+    certain_indices <- setdiff(1:nrow(x), uncertain_indices)
+    x <- x[certain_indices, ]
+  }
+  row.names(uncertain) <- NULL
+  row.names(x) <- NULL
+  structure(x, coord_uncertainity=uncertain)
```

Comment:
@vamsikrishna97 you label the rows pulled out as the `uncertain` object with `coord_uncertainity`, but in the examples above you do `attr(df, "coord_uncertain")` - 

the pattern so far has been to label the rows pulled out the same as the fxn name, so `coord_uncertain` instead of `coord_uncertainity`